core/biogeophys/cable_common.F90

```diff
@@ -693,12 +694,48 @@ subroutine get_unit(iunit)
 
   end subroutine get_unit
 
+  SUBROUTINE handle_iostat(ios, iomessage)
+    !*## Purpose
+    !
+    ! Take the return status of a read call and return an informative error
+    ! message on failure status.
+    !
+    !## Method
+    !
+    ! Use the IOSTAT and the new IOMSG specifier introduced in Fortran 2003
+    ! to throw an informative error message when the read errors.
+
+    INTEGER, INTENT(IN) :: ios
+    CHARACTER(LEN=*), INTENT(IN) :: ioMessage
+
+    IF (ios /= 0) THEN
+      ! Read of namelist failed
+      WRITE(ERROR_UNIT, FMT='(A)') ioMessage
+
+      ! This section was originally included, based on a method found [here]
+      ! (https://degenerateconic.com/namelist-error-checking.html), which would
+      ! also write the incriminating line to the error stream. However, it
+      ! relies on undefined behaviour, as the position in a file becomes
+      ! indeterminate if an error condition occurs during input/output (see
+      ! [this
+      ! thread](https://community.intel.com/t5/Intel-Fortran-Compiler/
+      ! Regression-with-quot-backspace-quot-in-2023-0/m-p/1448175)). While
+      ! it may work as desired for most compilers, it is not guaranteed, and
+      ! may change without warning at any time as evidenced by said thread.
+      !BACKSPACE(nmlUnit)
+      !READ(nmlUnit, FMT='(A)') BadLine
+      !WRITE(ERROR_UNIT, FMT='(A)') "Invalid line in namelist: "//TRIM(BadLine)
```

Comment:
```suggestion
```

I think it is safe to remove this comment as there will be no intention to uncomment the code in the future. This info would be best place either in the commit logs or the pull request description (which you have done already).

Reply:
Yep that was going to be my question, whether to leave this in there for posterity in case someone comes by in future and thinks "I know this other method, we should add that". As long as it's documented somewhere and findable I guess.